php/ui/component/class-debug.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Submit UI Component.
+ *
+ * @package Cloudinary
+ */
+
+namespace Cloudinary\UI\Component;
+
+use Cloudinary\UI\Component;
+use Cloudinary\Sync;
+use Cloudinary\Utils;
+
+/**
+ * Class Component
+ *
+ * @package Cloudinary\UI
+ */
+class Debug extends Component {
```

Comment:
<!-- highlight something positive -->
**👏 praise:** @DavidCramer, this is a great addition to the plugin.